upload/catalog/controller/extension/payment/coingate.php

```diff
@@ -127,12 +137,11 @@ public function callback()
         $this->response->addHeader('HTTP/1.1 200 OK');
     }
 
-    private function setupCoingateClient()
+    private function getCoingateClient()
```

Comment:
preset \CoinGate\Client::setAppInfo('OpenCart , ...) here , so it can be reused

Reply:
Done